app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -9,14 +9,46 @@ import MobileIntroStep from './mobile-intro-step';
 import DeviceContext from '../context/device';
 import ServiceProviderContext from '../context/service-provider';
 import Submission from './submission';
+import SubmissionStatus from './submission-status';
 import DesktopDocumentDisclosure from './desktop-document-disclosure';
 import useI18n from '../hooks/use-i18n';
+import { RetrySubmissionError } from './submission-complete';
+import SuspenseErrorBoundary from './suspense-error-boundary';
+import SubmissionInterstitial from './submission-interstitial';
 
 /** @typedef {import('react').ReactNode} ReactNode */
 /** @typedef {import('./form-steps').FormStep} FormStep */
 /** @typedef {import('../context/upload').UploadFieldError} UploadFieldError */
 
-function DocumentCapture() {
+/**
+ * Returns a new object with specified keys removed.
+ *
+ * @template {Record<string,any>} T
+ *
+ * @param {T} object Original object.
+ * @param {...string} keys Keys to remove.
+ *
+ * @return {Partial<T>} Object with keys removed.
+ */
+export const omit = (object, ...keys) =>
+  Object.entries(object).reduce((result, [key, value]) => {
+    if (!keys.includes(key)) {
+      result[key] = value;
+    }
+
+    return result;
+  }, {});
+
+/**
+ * @typedef DocumentCaptureProps
+ *
+ * @prop {boolean=} isAsyncPollingSubmission Whether submission should poll for async response.
```

Comment:
the "is" with "polling" makes me think this describes if it's currently polling, WDYT of naming this `shouldPollAsyncSumbission` or `isAsyncForm` or something like that?

Reply:
> the "is" with "polling" makes me think this describes if it's currently polling, WDYT of naming this `shouldPollAsyncSumbission` or `isAsyncForm` or something like that?

Sounds good to me 👍  `isAsyncForm` is also nice and concise, as an added bonus.

Reply:
> > the "is" with "polling" makes me think this describes if it's currently polling, WDYT of naming this `shouldPollAsyncSumbission` or `isAsyncForm` or something like that?
> 
> Sounds good to me 👍 `isAsyncForm` is also nice and concise, as an added bonus.

Renamed in 18cdf32.